ui/src/pages/search/authors/index.tsx

```diff
@@ -87,7 +87,7 @@ type Props = {
 const Authors: Types.NextPage<Props> = (props): React.ReactElement => {
     const router = Router.useRouter();
     // Query params
-    const [query] = React.useState(props.query ? props.query : '');
+    const { query = '' } = router.query as Interfaces.AuthorSearchQuery;
```

Comment:
Intriguing. So was this not defaulting to an empty string?

Reply:
Oh didn't notice you changed props.query to router.query

Reply:
Yeah, I think the issue was that the search form submit was only pushing the query term to router. So in order to get the SWR to re fetch we need to take the "query" var from the router instead of props. As it was, it worked after a refresh because getServerSideProps would get search results again based on the query param, but not from "in page" searches.